### PR TITLE
got first measurement flag

### PR DESCRIPTION
**Pull request by otim**
added flag to represent if the first measurement has arrived and accordingly a public member function to interface with

### CODE REVIEW

File: msf_core/include/msf_core/msf_sensorhandler.h [hunk 1]
```diff
@@ -52,10 +53,12 @@ class SensorHandler {
         manager_(mng),
         sensorID(constants::INVALID_ID),
         topic_namespace_(topic_namespace),
-        parameternamespace_(parameternamespace) {
+        parameternamespace_(parameternamespace),
+        received_first_measurement_(false) {
   }
   virtual ~SensorHandler() {
   }
+  bool receivedFirstMeasurement() {return received_first_measurement_;}
```
Comment (simonlynen) [inline msf_core/include/msf_core/msf_sensorhandler.h]: This method should be const

File: msf_updates/include/msf_updates/pose_sensor_handler/implementation/pose_sensorhandler.hpp [hunk 2]
```diff
@@ -124,6 +124,9 @@ void PoseSensorHandler<MEASUREMENT_TYPE, MANAGER_TYPE>::SetDelay(double delay) {
 template<typename MEASUREMENT_TYPE, typename MANAGER_TYPE>
 void PoseSensorHandler<MEASUREMENT_TYPE, MANAGER_TYPE>::ProcessPoseMeasurement(
     const geometry_msgs::PoseWithCovarianceStampedConstPtr & msg) {
+  if (!received_first_measurement_) {
```
Comment (simonlynen) [inline msf_updates/include/msf_updates/pose_sensor_handler/implementation/pose_sensorhandler.hpp]: Just set it to true without the additional branching

File: msf_updates/include/msf_updates/position_sensor_handler/implementation/position_sensorhandler.hpp [hunk 3]
```diff
@@ -83,6 +83,10 @@ void PositionSensorHandler<MEASUREMENT_TYPE, MANAGER_TYPE>::AdjustGPSZReference(
 template<typename MEASUREMENT_TYPE, typename MANAGER_TYPE>
 void PositionSensorHandler<MEASUREMENT_TYPE, MANAGER_TYPE>::ProcessPositionMeasurement(
     const sensor_fusion_comm::PointWithCovarianceStampedConstPtr& msg) {
+  if (!received_first_measurement_) {
```
Comment (simonlynen) [inline msf_updates/include/msf_updates/position_sensor_handler/implementation/position_sensorhandler.hpp]: Same here, just set to true

File: msf_updates/include/msf_updates/pressure_sensor_handler/implementation/pressure_sensorhandler.hpp [hunk 4]
```diff
@@ -43,6 +43,10 @@ void PressureSensorHandler::SetNoises(double n_zp) {
 
 void PressureSensorHandler::MeasurementCallback(
     const asctec_hl_comm::mav_imuConstPtr & msg) {
+  if (!received_first_measurement_) {
```
Comment (simonlynen) [inline msf_updates/include/msf_updates/pressure_sensor_handler/implementation/pressure_sensorhandler.hpp]: Same here and elsewhere

**Comment by simonlynen** [comment]
Thanks. Lgtm

File: msf_core/include/msf_core/msf_sensorhandler.h (hunk 1 above)
Comment (otim) [inline msf_core/include/msf_core/msf_sensorhandler.h]: I noticed that all (or most) methods in msf are CamelCased not camelCased... I will apply this here as well

**Pull request merged**